daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestCLIParsing.scala

```diff
@@ -716,4 +716,12 @@ class TestCLIparsing {
       cli.expectErr("Unable to load configuration")
     } (ExitCode.ConfigError)
   }
+
+  @Test def test_Layer_Execution_Error(): Unit = {
+    val schema = path("daffodil-cli/src/test/resources/org/apache/daffodil/layers/bad.dfdl.xsd")
+    runCLI(args"parse -s $schema") { cli =>
+      cli.sendLine("0", inputDone = true)
+      cli.expectErr("Unexpected exception in layer transformer 'bad'")
+    } (ExitCode.LayerExecutionError)
+  }
```

Comment:
I'd still like to see both a success test and a failure test to show that a 'buggy' layer transformer can perform work when it's not throwing an exception.  The good case could send 2, divide 10 by 2, and expect to see 5 while the bad case could send 0, divide 10 by 0, and expect an error message from the 'buggy' layer transformer.

Reply:
I've added a success test, renamed it to "buggy", and rebased for merging.